etc/liquibase/liquibase-maven-plugin.properties

```diff
@@ -0,0 +1,23 @@
+#
```

Comment:
This file should be ignored by the `.gitignore` in the latest develop

Reply:
I see you resolved this, but this file is still here. Did you forget to commit some changes?